onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -115,6 +119,26 @@ MIGraphXExecutionProvider::MIGraphXExecutionProvider(const MIGraphXExecutionProv
   if (!dump_model_ops_env.empty()) {
     dump_model_ops_ = (std::stoi(dump_model_ops_env) == 0 ? false : true);
   }
+
+  ROCBLAS_CALL_THROW(rocblas_create_handle(&external_rocblas_handle_));
+  ROCBLAS_CALL_THROW(rocblas_set_stream(external_rocblas_handle_, stream_));
+
+  MIOPEN_CALL_THROW(miopenCreate(&external_miopen_handle_));
+  MIOPEN_CALL_THROW(miopenSetStream(external_miopen_handle_, stream_));
+}
+
+MIGraphXExecutionProvider::~MIGraphXExecutionProvider() {
+  try {
+    ROCBLAS_CALL_THROW(rocblas_destroy_handle(external_rocblas_handle_));
```

Comment:
>ROCBLAS_CALL_THROW [](http://example.com/codeflow?start=4&length=18)

Is it guaranteed ROCBLAS_CALL_THROW will give out a std exception so we could capture? 
Further, maybe in destructor we should simply log the call error of "rocblas_destroy_handle" instead using ROCBLAS_CALL_THROW.

Reply:
it throws "ORT_THROW" just as CudaCall() does. 

Will follow the destructor of TensorRTExecutionProvider.

Reply:
Do not call ROCBLAS_CALL_THROW